var/spack/repos/builtin/packages/py-pyqtgraph/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyqtgraph(PythonPackage):
+    """PyQtGraph is a pure-python graphics and GUI library intended for use in mathematics,
+    scientific, and engineering applications"""
+
+    homepage = "http://www.pyqtgraph.org/"
+    pypi = "pyqtgraph/pyqtgraph-0.13.3.tar.gz"
+
+    license("MIT", checked_by="A-N-Other")
+
+    version("0.13.3", sha256="58108d8411c7054e0841d8b791ee85e101fc296b9b359c0e01dde38a98ff2ace")
+
+    depends_on("python@3.9:")
```

Comment:
```suggestion
    depends_on("python@3.8:", type=("build", "run"))
```
Could also remove it, Spack only supports Python 3.8+ anyway